### PR TITLE
Add the isogram exercise

### PR DESCRIPTION
**Pull request by jimmbraddock**

### CODE REVIEW

File: exercises/isogram/isogram_test.exs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+defmodule IsogramTest do
+  use ExUnit.Case
+
+  test "isogram in lowercase" do
+    refute Isogram.isogram?("eleven")
+  end
+
+  @tag :pending
+  test "not isogram in lowercase" do
```
Comment (parkerl) [inline exercises/isogram/isogram_test.exs]: Looks like the descriptions of this test and the one above are reversed?

File: exercises/isogram/isogram_test.exs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  end
+
+  @tag :pending
+  test "not isogram in upper and lower case" do
+    refute Isogram.isogram?("Alphabet")
+  end
+
+  @tag :pending
+  test "isogram with dash" do
+    assert Isogram.isogram?("thumbscrew-japingly")
```
Comment (parkerl) [inline exercises/isogram/isogram_test.exs]: If this test is driving out ignoring dashes then should the test case have multiple dashes?

File: exercises/isogram/isogram_test.exs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    refute Isogram.isogram?("the quick brown fox")
+  end
+
+  @tag :pending
+  test "isogram is phrase in upper and lower case" do
+    assert Isogram.isogram?("Emily Jung Schwartzkopf")
+  end
+
+  @tag :pending
+  test "not isogram with utf-8 letter" do
```
Comment (parkerl) [inline exercises/isogram/isogram_test.exs]: `not isogram with utf-8 letters`

File: exercises/isogram/isogram_test.exs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    assert Isogram.isogram?("Hjelmqvist-Gryb-Zock-Pfund-Wax")
+  end
+
+  @tag :pending
+  test "isogram with utf-8 letters in upper and lower case" do
+    assert Isogram.isogram?("Heizölrückstoßabdämpfung")
+  end
+
+  @tag :pending
+  test "phrase is not isogram" do
```
Comment (parkerl) [inline exercises/isogram/isogram_test.exs]: Should this be a test that spaces are ignored?

File: exercises/isogram/isogram_test.exs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    refute Isogram.isogram?("the quick brown fox")
+  end
+
+  @tag :pending
+  test "isogram is phrase in upper and lower case" do
+    assert Isogram.isogram?("Emily Jung Schwartzkopf")
```
Comment (parkerl) [inline exercises/isogram/isogram_test.exs]: If this is the test to show spaces are ignored then the case should not be varied.

File: exercises/isogram/isogram_test.exs (hunk 4 above)
Comment (jimmbraddock) [inline exercises/isogram/isogram_test.exs]: Yes. According with x-common the whitespaces should be ignored. I took the test data from x-common project, but couldn't come up nice description. Now I'm understand that some tests check several features at the same time.

File: exercises/isogram/isogram_test.exs (hunk 1 above)
Comment (jimmbraddock) [inline exercises/isogram/isogram_test.exs]: Yes, you are right. My bad english...

**Comment by jimmbraddock** [comment]
I changed tests according to principle "one test - one feature" and tried correctly change tests name.

**Comment by parkerl** [comment]
Great. I think this looks good. Anyone else want to 👍 this?

**Comment by dalexj** [comment]
👍

**Comment by parkerl** [comment]
Thanks :tada:

**Pull request merged**